pkg/test/runner/runner.go

```diff
@@ -142,17 +142,17 @@ func (r *Runner) runFnEval() error {
 		}
 	}
 	for i := 0; i < r.testCase.Config.RunCount(); i++ {
-		output, _, fnErr := runCommand("", r.kptBin, kptArgs)
+		stdout, stderr, fnErr := runCommand("", r.kptBin, kptArgs)
 		if fnErr != nil {
-			r.t.Logf("kpt error output: %s", output)
+			r.t.Logf("kpt error output: %s", stdout)
```

Comment:
We may need to print stderr here as well.

Reply:
fixed